docs/planning/backend/app/api/imports.py

```
@@ -0,0 +1,144 @@
+    """
+    Create the default family member reviewers.
+    """
+    from app.services import ReviewerService
+
+    default_names = ["Byron", "Veronica", "Bayden", "Ariannah"]
+    created = []
+
+    for name in default_names:
+        existing = ReviewerService.get_by_name(db, name)
+        if not existing:
+            reviewer = ReviewerService.get_or_create(db, name)
```

Ruff (0.14.10)

Local variable `reviewer` is assigned to but never used

Remove assignment to unused variable `reviewer`

(F841)